.github/workflows/ci.yml

```diff
@@ -9,8 +9,27 @@ on:
       - main
 
 jobs:
-  lint-build-and-test:
-    name: Lint, build and test (${{ matrix.os }})
+  rustfmt:
+    name: RustFmt Check
+    runs-on: ubuntu-latest
+    timeout-minutes: 45
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Install Rust toolchain
+        uses: actions-rust-lang/setup-rust-toolchain@v1
+        with:
+          components: rustfmt
+          rustflags: ""
+
+      - name: Check license headers
+        run: ./tools/scripts/check-license-headers
+
+      - name: Rustfmt Check
+        uses: actions-rust-lang/rustfmt@v1
```

Comment:
Is the format check delaying the previous linting step too much? I thought this was always quite fast.

Reply:
No but it's nice to have them concurrent and it makes it an individual check on the PR checks.